test/jdk/java/awt/print/Dialog/DialogOrient.java

```diff
@@ -69,8 +69,8 @@ private static void init() throws Exception {
       Sysout.printInstructions( instructions );
 
         PrinterJob job = PrinterJob.getPrinterJob();
-        job.setPrintable(new DialogOrient());
         PageFormat landscape = job.pageDialog(job.defaultPage());
+        job.setPrintable(new DialogOrient(), landscape);
```

Comment:
why doesn't job.setPrintable(DialogOrient) work here? doesn't job.pageDialog already set the job orientation to landscape?

Reply:
https://docs.oracle.com/en/java/javase/20/docs/api/java.desktop/java/awt/print/PrinterJob.html#setPrintable(java.awt.print.Printable)
says "The [PageFormat](file:///D:/jdk-17.0.1_doc-all/docs/api/java.desktop/java/awt/print/PageFormat.html) for each page is the default page format."
which is Portrait so it seems prudent to pass the pageformat modified in pageDialog to setPrintable

Reply:
@prsadhuk 

The test works with the fix. The redundant if block can be removed, it is repeated twice in the test Ln#78.

```
 if (job.printDialog()){
            job.print();
        }
```